sql/2020/22_HTTP_2/measure_number_of_tcp_connections_per_site.sql

```diff
@@ -0,0 +1,41 @@
+# standardSQL
+# Measure number of TCP Connections per site.
+SELECT 
+  "mobile" AS client,
+  JSON_EXTRACT_SCALAR(payload, "$._protocol") AS protocol,
+  COUNT(0) AS num_pages,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(50)] AS median,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(75)] AS p75,
+  APPROX_QUANTILES(_connections, 100)[SAFE_ORDINAL(95)] AS p95
+FROM 
+  `httparchive.requests.2020_08_01_mobile` AS requests
+INNER JOIN
+  `httparchive.summary_pages.2020_08_01_mobile` AS summary
+ON
+  requests.url = summary.url
+WHERE
+  JSON_EXTRACT_SCALAR(payload, "$._is_base_page") = "true"
```

Comment:
Should we use `firstHTML` from `httparchive.almanac.requests` table? More consistent with other queries.

Reply:
Agreed that makes more sense. I'll update the query

Reply:
@rviscomi modified the query and i just committed the suggestion